modules/configuring-firewall.adoc

```diff
@@ -92,11 +96,23 @@ CDN hostnames, such as `cdn01.quay.io`, are covered when you add a wildcard entr
 |443, 80
 | Required to access your GCP account.
 
-|Azure
+.4+|Azure
 |`management.azure.com`
 |443, 80
 |Required to access Azure services and resources. Review the link:https://docs.microsoft.com/en-us/rest/api/azure/[Azure REST API Reference] in the Azure documentation to determine the endpoints to allow for your APIs.
 
+|`*.blob.core.windows.net`
+|443, 80
+|Required to download ignition files.
```

Comment:
Ignition is a productized utility so we should capitalize it ("Ignition files").